<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PtKeyPointSIFTClass</class>
 <widget class="QDialog" name="PtKeyPointSIFTClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>157</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SIFT关键点</string>
  </property>
  <property name="windowIcon">
   <iconset resource="LidarMaster.qrc">
    <normaloff>:/LidarMaster/img/logo.png</normaloff>:/LidarMaster/img/logo.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>阈值:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ld_val"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>尺度个数:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ld_num"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>尺度标准差:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ld_std"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>尺度空间层次:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ld_level"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pb_SIFT_OK">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="LidarMaster.qrc"/>
 </resources>
 <connections/>
</ui>
